data/abrokinla.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Araoye Abraham A.",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Teacher turnerd **Web Developer**. Motivatied to ease human endeavour by code",
```

Comment:
There is a little *misspelling*. I think the word **"turnerd"** here should be **"turned"** instead.  And also, the word **"Motivatied"** should be **"Motivated"**.

Reply:
Good spot Abbey 👍 

@abrokinla These can come in another pull request, I will merge this now. Here are the docs for editing your profile https://linkfree.eddiehub.io/docs/how-to-guides/editing